{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/tests/test_admin.py

```diff
@@ -47,3 +53,25 @@ def test_view_user(self, admin_client):
         url = reverse("admin:users_user_change", kwargs={"object_id": user.pk})
         response = admin_client.get(url)
         assert response.status_code == 200
+
+    @pytest.fixture
+    def force_allauth(self, settings):
+        settings.DJANGO_ADMIN_FORCE_ALLAUTH = True
+        # Reload the admin module to apply the setting change
+        import {{ cookiecutter.project_slug }}.users.admin as users_admin  # pylint: disable=import-outside-toplevel
+
+        try:
+            reload(users_admin)
+        except admin.sites.AlreadyRegistered:
+            pass
+
+    @pytest.mark.django_db
+    @pytest.mark.usefixtures("force_allauth")
+    def test_allauth_login(self, rf, settings):
+        request = rf.get("/fake-url")
+        request.user = AnonymousUser()
+        response = admin.site.login(request)
+
+        # The `admin` login view should redirect to the `allauth` login view
+        target_url = reverse(settings.LOGIN_URL) + "?next=" + request.path
+        assertRedirects(response, target_url, fetch_redirect_response=False)
```

Comment:
I had to use `fetch_redirect_response=False` since the response would need a client attribute otherwise. I can add `response.client = Client()` if fetching the redirect is important.